docs/samples/contact-center/index.html

```diff
@@ -85,19 +85,46 @@ <h2 class="collapsible">
   </section>
   </div>
 
-  <div class="box">
-    <section class="section-box">
-      <h2 class="collapsible">
-        Agent Desktop Using Webex CC SDK
-        <i class="arrow fa fa-angle-down" aria-hidden="true"></i>
-      </h2>
-
-      <div class="section-content">
+    <div class="box">
+      <section class="section-box">
+        <h2 class="collapsible">
+          Agent Desktop Using Webex CC SDK
+          <i class="arrow fa fa-angle-down" aria-hidden="true"></i>
+        </h2>
+  
+        <div class="section-content">
+          <fieldset>
+            <legend>Agent</legend>
+            <p class="note">
+              NOTE: Teams are fetched automatically for the Agent Login.
```

Comment:
How is this going to happen? Could you please point me towards the code where we have taken care of fetching the team automatically or is that information supposed to come from agent profile